### ISSUE TITLE
No possibility to change the background color of the whole software ?

### ISSUE THREAD

**Issue by Martyprod** [Xyrio/RSSOwlnix#103]
Hello

is there a way to change the color of Rssowlnix ? it was not possible with RSSOwl et it's a big problem for me, i have have eyes problems, and the full background white is not possible for me to use anymore. Is it possible to change that , and have text in white and background color in an another color than white ?

**Comment by Xyrio** [comment]
later eclipse versions can but im not sure if that automatically works. currently i have not updated to latest because they dropped 32bit support and around 30% of windows downloads are still for 32bit.

**Comment by Martyprod** [comment]
Hey Xyrio, thanks for the reply. unfortunately i changed my screen from an old CRT to a new LED one, and even with the brighness to zero, the white background is no more possible for me. 
i'll wait with "impatience" a possibility. 
i'm using an awful online service (inor****) waiting a possibility to change the background white color... Thanks to develop this fork. i love this software used the original one since a very very long time and hope it will only a temporary good bye !

**Comment by pchris7489** [comment]
Wełl, this is the main reason I totally abandoned RSSOwl. It was becoming very difficult to stand this white consistent background late at night an with no lights.

**Comment by Martyprod** [comment]
yes, i switched to Inoreader,i don't like it because there is no privacy, but didn't found an alternative, it's sad for RSSwl..
